scripts/testnet/variables.sh

```diff
@@ -65,8 +65,6 @@ export OBSERVERS_ANTIFLOOD_DISABLE=0
 # For now, make sure that you checkout feat/sovereign branch from mx-chain-deploy repo when using these scripts
 export SOVEREIGN_DEPLOY=1
 
-export ROUND_DURATION=4000
```

Comment:
If you removed this, you should remove its usage from entire project. Remove if as well from `scripts/testnet/include/config.sh`